apps/voting/contracts/Voting.sol

```diff
@@ -73,6 +96,9 @@ contract Voting is IForwarder, AragonApp {
     event ChangeMinQuorum(uint64 minAcceptQuorumPct);
     event ChangeVoteTime(uint64 voteTime);
     event ChangeObjectionPhaseTime(uint64 objectionPhaseTime);
+    event SetDelegate(address indexed voter, address indexed delegate);
+    event ResetDelegate(address indexed voter, address indexed delegate);
```

Comment:
the same as fro method `ResetDelegate` it's also better to rename the argument `delegate` to `removedDelegated` or so

Reply:
Thanks for pointing that out; I'll come up with more appropriate names.

Reply:
Fixed in 94e327614f7ec8785b6fe9cf4805150877a679d6.